src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -92,6 +92,9 @@
 
          Available options:
          "reboot" (type 'b')
+         "install-packages" (type 'as')
+         "uninstall-packages" (type 'as')
+         "install-local-packages" (type 'ah')
```

Comment:
Oh I see, so we had different views of what "unified" meant. 🔀
Hmm, yeah I think that makes sense.

If we do that, then I'd like to back out the `install-local-packages` option
from `PkgChange` and make it use that instead as well. It just felt weird for it to be in the options variant.

It almost feels like to keep this as generic as possible, we want two variants, i.e. an origin-based variant, which contains the info to modify the deployment's new origin, and a transaction-based variant, which contains flags relevant only in the scope of the transaction. So maybe something like:

```xml
<!-- Available origin keys:
     "refspec" (type 's')
     "revision" (type 's')
     "install-packages" (type 'as')
     "uninstall-packages" (type 'as')
     "install-local-packages" (type 'ah')

     Eventually, also:
     "regenerate-initramfs" (type 'b')
     "initramfs-args" (type 'as')
-->
<!-- Available options keys:
     "reboot" (type 'b')
     "allow-downgrade" (type 'b')
     "skip-purge" (type 'b')
     "dry-run" (type 'b')
-->
<method name="CreateDeployment">
  <arg type="a{sv}" name="origin" direction="in"/>
  <arg type="a{sv}" name="options" direction="in"/>
  <arg type="s" name="transaction_address" direction="out"/>
  <annotation name="org.gtk.GDBus.C.UnixFD" value="true"/>
</method>
```

?